layouts/partials/scriptsBodyStart.html

```diff
@@ -1,2 +1,7 @@
-{{ $script_main := resources.Get "js/themeLoader.js" | minify | fingerprint }}
-<script src="{{ $script_main.RelPermalink }}"></script>
+{{ if hugo.IsProduction }}
+{{ $theme_script := resources.Get "js/theme.js" | minify | fingerprint }}
+<script src="{{ $theme_script.RelPermalink }}" integrity="{{ $theme_script.Data.Integrity }}"></script>
+{{ else }}
+{{ $theme_script := resources.Get "js/theme.js" | fingerprint }}
```

Comment:
@kusyka911, do you think `| fingerprint` is useful, even when we are not using the `integrity` attribute in the script tag?

Reply:
Fingerprint will be added to filename. This will prevent JS caching by browser while you running website in development mode and making JS changes.

Reply:
Thanks for your response @kusyka911 .

> This should prevent JS caching by browser 

I have another perspective though:

- If someone in concerned about caching, most likely they will disable by going it to Dev tools. This is because they are concerned not only about JS caching but everything else (CSS, other assets) as well. The computation involved in calculating hash (due to `| fingerprint`) is rendered useless in this case.
Considering this, the wastage of computation can be quite high In development environment since, file changes (and thus hash computation) can happen very frequently.
- If someone is **not** concerned about caching, the computation involved in calculating hash is anyways not going to be useful.

What do you have to say in this regard?

> P.S. I'm sorry for late response, right now I'm a little busy working on another small project.

No worries. That's aright.

Reply:
I have proceeded with removing ` fingerprint` in development. However, please feel free to submit a PR, if you see any good use case.